docs/migrating/backup.md

```diff
@@ -0,0 +1,59 @@
+# Backup 
+
+You should always make sure that your solution is properly backed up. The following example shows you how to do this on a Linux/UNIX based system where eZ Platform is using a MySQL database called "example". You should close the system before making a backup or an upgrade.
+
+!!! note "Externally stored assets"
+
+    If you store assets in any external service or localization, you should back them up before proceeding.
+
+1\. Navigate into the eZ Platform directory:
+ 
+```
+$ cd /path/to/ezplatform
+```
+
+2\. Unclusterize or backup the NFS folder.
+ 
+3\. Clear all caches:
+
+```
+$ var/cache/*
+$ var/logs/*
+$ var/<var_dir>/logs/*
+$ var/<var_dir>/cache/*
+```
+
+4\. Create a dump of the database:
+ 
+```
+$ mysqldump -u root --add-drop-table example > db_backup.sql
+```
+
+5\. In parent directory create a tape archive of the files (including the DB dump) using the "tar" command:
```

Comment:
Not sure if "tape archive" is used as a full term, I think "tar" is more common and less confusing.